cmd/cycloid/stacks/create_from_blueprint.go

```diff
@@ -0,0 +1,114 @@
+package stacks
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/cycloidio/cycloid-cli/cmd/cycloid/common"
+	"github.com/cycloidio/cycloid-cli/cmd/cycloid/middleware"
+	"github.com/cycloidio/cycloid-cli/internal/cyargs"
+	"github.com/cycloidio/cycloid-cli/printer"
+	"github.com/cycloidio/cycloid-cli/printer/factory"
+)
+
+// CreateFromBlueprintOutput represents the formatted output for create-from-blueprint command
+type CreateFromBlueprintOutput struct {
+	Canonical   string `json:"canonical"`
+	Name        string `json:"name"`
+	Ref         string `json:"ref"`
+	Description string `json:"description"`
+}
+
+func NewCreateFromBlueprintCommand() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:     "create-from-blueprint",
+		Short:   "create a new stack from a blueprint",
+		Example: `cy --org my-org stack create-from-blueprint --blueprint-ref repo:blueprint-canonical --name "My Stack" --canonical my-stack --service-catalog-source-canonical my-catalog --use-case production`,
+		RunE:    createFromBlueprint,
```

Comment:
Please, specify to cobra if your command accept no arguments, that allow cobra to be more consistent in shell completion.
```suggestion
		RunE:    createFromBlueprint,
		Args:    cobra.NoArgs,
```